include/na/zoned/Compiler.hpp

```diff
@@ -243,10 +237,21 @@ class Compiler : protected Scheduler,
   }
 };
 
+class RoutingAgnosticSynthesizer
+    : public PlaceAndRouteSynthesizer<RoutingAgnosticSynthesizer,
+                                      VertexMatchingPlacer,
+                                      IndependentSetRouter> {
+public:
+  RoutingAgnosticSynthesizer(const Architecture& architecture,
+                             const Config& config)
+      : PlaceAndRouteSynthesizer(architecture, config) {}
+  explicit RoutingAgnosticSynthesizer(const Architecture& architecture)
+      : PlaceAndRouteSynthesizer(architecture) {}
```

Comment:
This is just a nitpick, but I think this can be a little shorter and more concise via something along the lines of
```
  using PlaceAndRouteSynthesizer::PlaceAndRouteSynthesizer;
```
Instead of explicitly redefining the constructors. Also holds below.